src/hooks.tsx

```diff
@@ -44,7 +44,19 @@ export const useDisableBodyScroll = (disable: boolean): void => {
                     // if the scrollbar is visible, we don't want to hide it because content will be resized
                     const overflowY = hasScrollbar ? 'scroll' : 'hidden';
 
-                    document.body.style.cssText = `overflow:hidden;overflow-y:${overflowY};position:fixed;top:${-bodyScrollTop}px;left:0px;right:0px;bottom:0px`;
+                    document.body.style.cssText =
+                        bodyStyles +
+                        (bodyStyles.endsWith(';') ? '' : ';') +
+                        [
+                            'overflow: hidden;',
+                            `overflow-y: ${overflowY};`,
+                            'position: fixed;',
+                            `top: ${-bodyScrollTop}px;`,
+                            'left: 0px;',
+                            'right: 0px;',
+                            'bottom: 0px;',
+                            'overscroll-behavior-y: contain;', // disable overscroll
+                        ].join('');
```

Comment:
This keeps previous body styles instead of replacing them

Reply:
maybe.
I've done some manual testing and everything seems to work fine